templates/taxi/manufacturer_list.html

```diff
@@ -1,33 +1,39 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
+  <h1>Manufacturer List</h1>
+  <a href="{% url 'taxi:manufacturer-create' %}" class="btn btn-primary">Add manufacturer</a>
+  {% if manufacturer_list %}
+    <table class="table">
+      <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Country</th>
+        <th>Upgrade</th>
```

Comment:
The table header 'Upgrade' might be misleading. Consider changing it to 'Update' to match the action performed by the link, which is to update the manufacturer's details.